Storage/SQL/BrowserDB.swift

```diff
@@ -100,22 +118,19 @@ open class BrowserDB {
     // Checks if the database schema needs created or updated and acts accordingly.
     // Calls to this function will be serialized to prevent race conditions when
     // creating or updating the schema.
-    func prepareSchema(_ schema: Schema) -> DatabaseOpResult {
+    fileprivate static func prepareSchema(_ schema: Schema, forDB db: SwiftData, withConnection connection: SQLiteDBConnection) -> DatabaseOpResult {
         SentryIntegration.shared.addAttributes(["dbSchema.\(schema.name).version": schema.version])
 
         // Ensure the database has not already been closed before attempting to
-        // create or update the schema.
+        // create or update the schema. This *should* never happen since an open
+        // connection should always be passed in, but check to be safe.
         guard !db.closed else {
```

Comment:
I'm not sure this is necessary. It might be here so that we don't try to init a connection that we fetched while we were shutting down.